flink-dist/src/main/assemblies/bin.xml

```diff
@@ -38,10 +38,9 @@ under the License.
 			<useTransitiveFiltering>true</useTransitiveFiltering>
 
 			<includes>
-				<include>org.apache.logging.log4j:log4j-api</include>
-				<include>org.apache.logging.log4j:log4j-core</include>
-				<include>org.apache.logging.log4j:log4j-slf4j-impl</include>
-				<include>org.apache.logging.log4j:log4j-1.2-api</include>
+				<include>org.slf4j:log4j-over-slf4j</include>
+				<include>ch.qos.logback:logback-classic</include>
+				<include>ch.qos.logback:logback-core</include>
```

Comment:
This file indicates that the jars should not be in the fat jar. We now want the logback jars to not be in the fat jar. Potentially not neccessary now that the log jars are the ones we want. But nice to keep the same pattern for our logging library.